### DISCUSSION TITLE
pyqt

### DISCUSSION THREAD

**Discussion by mahesh11T** [zauberzeug/nicegui#1306] [Q&A] [answered]
### Question

Is it possible to bring a existing Pyqt app into nicegui framework? , I know we can only make desktop application with pqyt, just wanted to know if any wrapper is available which maps pyqt elements with nicegui's.

**Comment by rodja** [comment, marked as answer]
Not that I'm aware of. But that sounds like an interesting project. Would you like to try implementing it?

**Reply by mahesh11T** [reply]
I would love to, but I am busy with some work.

**Reply by WorkSync** [reply]
> I would love to, but I am busy with some work.我很想去，但我正忙于一些工作。

I am working on  this.Let me try and contribute.

**Comment by rodja** [comment]
If you just want to show a NiceGUI page inside PyQt5 application, you could probably use it's webview and point it to the url of NiceGUI. You can get it with [`app.urls`](https://nicegui.io/documentation/section_configuration_deployment#urls) programatically. We do the same for NiceGUI [Native Mode](https://nicegui.io/documentation/section_configuration_deployment#native_mode).
